src/js/MainController.js

```diff
@@ -394,11 +394,14 @@ goog.require('ga_topic_service');
     }
 
     // An new appcache file is available.
-    if ($window.applicationCache) { // IE9
+    if ($window.applicationCache) {
       $window.applicationCache.addEventListener('obsolete', function(e) {
-        $window.location.reload();
+        setTimeout(function() {
```

Comment:
Yes. It's the fix for Firefox. Without this, the reload does happend, but the appcache update results in an error (thus requireing another manual reload).

With the timeout, it works directly.